build/install-dotnet.ps1

```diff
@@ -0,0 +1,8 @@
+# Installs .NET Core 2.1 and .NET 6 for CI/CD environment
+# see: https://docs.microsoft.com/en-us/dotnet/core/tools/dotnet-install-script#examples
+
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12;
+
+&([scriptblock]::Create((Invoke-WebRequest -UseBasicParsing 'https://dot.net/v1/dotnet-install.ps1'))) -Version 2.1.816
```

Comment:
> -Version 2.1.816

Why do we need version `2.1.816`?

Reply:
Tests still build with .NET Core 2.1 for the time being to make sure we exercise the netstandard 2.0 build.